### PR TITLE
chore: Add 3.8.0, remove 3.4.1, 3.6.1, 3.6.2

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
Part of https://github.com/stackabletech/issues/issues/626

```[tasklist]
### Dependencies
- [ ] https://github.com/stackabletech/docker-images/pull/813
```

> [!IMPORTANT]
> I have left 3.7.1 as LTS as per spreadsheet, and therefore left the `kafka-latest` test dimension and getting_started version as is.

### CODE REVIEW

File: tests/test-definition.yaml
Comment (sbernauer) [inline tests/test-definition.yaml:13]:
```suggestion
      - 3.7.1 # Using LTS version here
```

**Review by sbernauer — DISMISSED** [review]
LGTM when tests pass

**Comment by NickLarsenNZ** [comment]
> LGTM when tests pass

Tests passed in https://github.com/stackabletech/docker-images/pull/813

**Comment by sbernauer** [comment]
Yeah, but this test will fail in CI before we merge https://github.com/stackabletech/docker-images/pull/813, is that correct?

**Review by sbernauer — APPROVED** [review]

**Pull request merged**
